lib/api_browser/app/index.html

```diff
@@ -14,35 +14,8 @@
   <div ui-view=""></div>
   <!-- build:js({.tmp,app}) scripts/praxis.js -->
   <!-- bower:js -->
-  <script src="bower_components/jquery/dist/jquery.js"></script>
-  <script src="bower_components/angular/angular.js"></script>
-  <script src="bower_components/lodash/dist/lodash.compat.js"></script>
-  <script src="bower_components/angular-ui-router/release/angular-ui-router.js"></script>
-  <script src="bower_components/angular-ui-bootstrap-bower/ui-bootstrap-tpls.js"></script>
-  <script src="bower_components/angular-sanitize/angular-sanitize.js"></script>
   <!-- endbower -->
   <!-- fileblock:js scripts -->
-  <script src="js/app.js"></script>
-  <script src="js/filters/resource_name.js"></script>
-  <script src="js/controllers/menu.js"></script>
-  <script src="js/controllers/action.js"></script>
-  <script src="js/directives/attribute_table.js"></script>
-  <script src="js/controllers/type.js"></script>
-  <script src="js/controllers/controller.js"></script>
-  <script src="js/factories/Documentation.js"></script>
-  <script src="js/directives/type_label.js"></script>
-  <script src="js/filters/attribute_name.js"></script>
-  <script src="js/filters/friendly_json.js"></script>
-  <script src="js/directives/attribute_description.js"></script>
-  <script src="js/directives/attribute_table_row.js"></script>
-  <script src="js/factories/TypeTemplates.js"></script>
-  <script src="js/directives/no_container.js"></script>
-  <script src="js/filters/is_empty.js"></script>
-  <script src="js/directives/request_body.js"></script>
-  <script src="js/factories/PayloadTemplates.js"></script>
-  <script src="js/factories/TemplateProvider.js"></script>
-  <script src="js/directives/request_headers.js"></script>
-  <script src="js/directives/request_parameters.js"></script>
```

Comment:
Yeah, they will get overwritten, but the explicit order here defined is preserved.